catalog-scripts/idp-catalog-wizard.py

```diff
@@ -0,0 +1,174 @@
+import argparse
+import requests
+import os
+# import yaml
+import re
+from requests.auth import HTTPBasicAuth
+import argparse
+from requests.packages.urllib3.util.retry import Retry
+from requests.adapters import HTTPAdapter
+import subprocess
+
+current_directory = os.path.basename(os.getcwd())
+yaml_content_template = """
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  tags:
+    - java
+    - harness
```

Comment:
```suggestion
```

Reply:
@ajinkya-harness Are we removing these tags?